install_deps.sh

```diff
@@ -41,7 +40,8 @@ apt-get update && apt-get ${APT_GET_FLAGS} install \
     sudo \
     vim \
     iproute2 \
-    wget
+    wget \
+    libhiredis-dev
```

Comment:
Does ccache use redis?

Reply:
This is for future use - secondary cache.Rethinking - I'm going to remove it for now. If we want to use secondary redis cache in the future, we just add the support.

Reply:
removed this installed.